mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -910,6 +910,7 @@ void CMomentumPlayer::OnZoneEnter(CTriggerZone *pTrigger)
                     const auto pLauncher = static_cast<CMomentumStickybombLauncher *> (GetWeapon(WEAPON_STICKYLAUNCHER));
                     if (pLauncher)
                     {
+                        pLauncher->StopChargeSound();
```

Comment:
This should be done in the `SetChargeEnabled` method since it is only being ran bundled with it, making it a loose dependency. I'd rather only have to call `SetChargeEnabled(false)` instead of having to remember to also call `StopChargeSound` as well, if that makes sense.

Reply:
Had it there before but opted for a cleaner `SetChargeEnabled` function. Will change it back.